packages/abi-typegen/src/utils/parseTypeArguments.test.ts

```diff
@@ -108,7 +108,7 @@ describe('parseTypeArguments.ts', () => {
   test('should fallback to void for null outputs', () => {
     const project = getTypegenForcProject(AbiTypegenProjectsEnum.FN_VOID);
 
-    const types = bundleTypes([]);
+    const types = bundleTypes(project.abiContents.types);
```

Comment:
Why was this necessary?

Reply:
Because this test is testing that the output is `void`, which is now [handled](https://github.com/FuelLabs/fuels-ts/blob/0a7cf759cac2d2e934be714654852e7037b8c700/packages/abi-typegen/src/abi/types/EmptyType.ts#L21) by the `EmptyType` and so we need the empty type in the types array:
```ts
[
    {
      "typeId": 0,
      "type": "()",
      "components": [],
      "typeParameters": null
    }
]
```
This type comes included in `project.abiContents.types`.